easygui/boxes/button_box.py

```diff
@@ -408,6 +408,8 @@ def create_msg_widget(self, msg):
             state=tk.DISABLED,
             padx=(global_state.default_hpad_in_chars) *
             self.calc_character_width(),
+            relief="flat",
+            background=self.boxRoot.config()["background"][-1],
```

Comment:
This is the platform-compatible solution. Takes the background value from the boxRoot.